examples/cms-contentstack/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Examples don't have a license, please delete this file.

Reply:
remove license file